livedata/src/test/java/androidx/arch/core/executor/testing/InstantExecutorExtension.kt

```diff
@@ -0,0 +1,25 @@
+package androidx.arch.core.executor.testing
+
+import androidx.arch.core.executor.ArchTaskExecutor
+import androidx.arch.core.executor.TaskExecutor
+import org.junit.jupiter.api.extension.AfterEachCallback
+import org.junit.jupiter.api.extension.BeforeEachCallback
+import org.junit.jupiter.api.extension.ExtensionContext
+
+class InstantExecutorExtension : BeforeEachCallback, AfterEachCallback {
```

Comment:
Nicely done 👍

Reply:
I took it from view models library. One point more to keep sources in one repo

Reply:
[Testfixture](https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures) is a great feature that Gradle provides us to handle it, but it is not supported by AGP yet :cry:  ([Jetbrains issue](https://youtrack.jetbrains.com/issue/KT-33877), [AGP Issue](https://issuetracker.google.com/issues/139438142), [AS Issue](https://issuetracker.google.com/issues/139762443))

Reply:
^ I think this comment is on the wrong discussion thread 🙂